### PR TITLE
Dat 3523 | add styles for portable tables 

### PR DESCRIPTION
**Pull request by RafLeszczynski**
https://wikia-inc.atlassian.net/browse/DAT-3523

### CODE REVIEW

**Comment by RafLeszczynski** [comment]
@rogatty, @hakubo, @vforge, @bognix feel free to take a look :-)

File: front/styles/main/module/article/_portable-table.scss [hunk 1]
```diff
@@ -0,0 +1,53 @@
+$portable-table-cell-horizontal-padding: 56px;
```
Comment (dianafa) [inline front/styles/main/module/article/_portable-table.scss]: :O magical number?

**Comment by dianafa** [comment]
LGTM

File: front/styles/main/module/article/_portable-table.scss (hunk 1 above)
Comment (RafLeszczynski) [inline front/styles/main/module/article/_portable-table.scss]: it's a padding used defined in Material UI, and I moved it to the variable so it's not so magic :-)

Comment (vforge) [inline front/styles/main/module/article/_portable-table.scss]: If it's not re-used, there's no need to var it ;)

Comment (RafLeszczynski) [inline front/styles/main/module/article/_portable-table.scss]: Sass Linter complains :-)

**Comment by vforge** [comment]
LGTM

File: front/scripts/main/components/article-content.js [hunk 2]
```diff
@@ -460,7 +460,13 @@ export default App.ArticleContentComponent = Ember.Component.extend(
 		handleTables() {
 			this.$('table:not([class*=infobox], .dirbox)')
 				.not('table table')
-				.wrap('<div class="article-table-wrapper"/>');
-		},
+				.each((index, element) => {
+					const $element = this.$(element),
+						wrapper = `<div class="article-table-wrapper${$element.data('portable') ?
```
Comment (hakubo) [inline front/scripts/main/components/article-content.js]: I would extract that for readability
also
element.getAttribute('data-portable')- will be much performant here

**Comment by vforge** [comment]
LGTM

File: front/styles/main/module/article/_portable-table.scss (hunk 1 above)
Comment (vforge) [inline front/styles/main/module/article/_portable-table.scss]: We'll tackle that later ;) Thanks!

File: front/scripts/main/components/article-content.js (hunk 2 above)
Comment (RafLeszczynski) [inline front/scripts/main/components/article-content.js]: @hakubo what about using https://www.npmjs.com/package/classnames for such a case?

**Comment by idradm** [comment]
LGTM

**Pull request merged**